### PR TITLE
BRS-409 footer link cards implemented

### PR DESCRIPTION
**Pull request by DayKirby-OXD**
### Jira Ticket:
BRS-409

### Jira Ticket URL:
https://bcparksdigital.atlassian.net/browse/BRS-409

### Description:
Implement footer variation of LinkCard component
Implemented for Reconciliation page, content updates are detailed in ticket
Line break changes in linkCard.scss and index.js are unintentional but harmless

### CODE REVIEW

File: src/staging/src/components/pageContent/linkCard.js [hunk 1]
```diff
@@ -34,13 +34,12 @@ export default function LinkCard({ url, title, subTitle, imageUrl, imageAltText,
       cardClasses += ""
       break;
   }
-
   return (
     <div className={wrapClasses}>
     <Link
```
Comment (BcGovNeal) [inline src/staging/src/components/pageContent/linkCard.js:39]: Getting warnings in the console about using the `Link` component for external URLs.   May want to create a link wrapper component to switch between `Link` and regular anchor elements

**Review by BcGovNeal — COMMENTED** [review]
On a first look the output doesn't match the D

File: src/staging/src/components/pageContent/linkCard.js (hunk 1 above)
Comment (DayKirby-OXD) [reply src/staging/src/components/pageContent/linkCard.js:39]: Interesting. I'll touch base with Cole on that as he introduced that in a prev branch

Comment (cole) [reply src/staging/src/components/pageContent/linkCard.js:39]: Creating a switch component is probably the correct solution. In the short term we could switch to just using a tags, it's a bit slower with internal links but still works.

Comment (DayKirby-OXD) [reply src/staging/src/components/pageContent/linkCard.js:39]: will switch to <a> for this ticket, and add a tech debt ticket to look into `Link`

File: src/staging/src/components/pageContent/linkCard.js [hunk 2]
```diff
@@ -24,8 +23,8 @@ export default function LinkCard({ url, title, subTitle, imageUrl, imageAltText,
       cardClasses += "link-card--horz"
       break;
     case "Footer": // TODO for inner pages
```
Comment (cole) [inline src/staging/src/components/pageContent/linkCard.js:25]: Remove TODO?

**Review by cole — APPROVED** [review]
Looks good

**Review by BcGovNeal — APPROVED** [review]

File: src/staging/src/components/pageContent/linkCard.js (hunk 2 above)
Comment (DayKirby-OXD) [reply src/staging/src/components/pageContent/linkCard.js]: removed.

**Pull request merged**